sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/application-changes-report-integration/_tests_/application-changes-report-integration.scheduler.e2e-spec.ts

```diff
@@ -0,0 +1,383 @@
+import { DeepMocked } from "@golevelup/ts-jest";
+import { INestApplication } from "@nestjs/common";
+import {
+  QueueNames,
+  addDays,
+  getISODateOnlyString,
+  getPSTPDTDateTime,
+} from "@sims/utilities";
+import {
+  createTestingAppModule,
+  describeProcessorRootTest,
+  mockBullJob,
+} from "../../../../../../test/helpers";
+import { ApplicationChangesReportIntegrationScheduler } from "../application-changes-report-integration.scheduler";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeEducationProgramOffering,
+  createFakeStudent,
+  createFakeStudentAssessment,
+  createFakeUser,
+  saveFakeApplicationDisbursements,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { getUploadedFile } from "@sims/test-utils/mocks";
+import * as Client from "ssh2-sftp-client";
+
+import {
+  Application,
+  ApplicationStatus,
+  AssessmentTriggerType,
+  COEStatus,
+  DisbursementScheduleStatus,
+  EducationProgramOffering,
+  OfferingIntensity,
+  Student,
+  User,
+} from "@sims/sims-db";
+import { createFakeSINValidation } from "@sims/test-utils/factories/sin-validation";
+import { In, IsNull, Not } from "typeorm";
+import {
+  ApplicationChangesReportHeaders,
+  APPLICATION_CHANGES_DATE_TIME_FORMAT,
+} from "@sims/integrations/esdc-integration";
+
+describe(
+  describeProcessorRootTest(QueueNames.ApplicationChangesReportIntegration),
+  () => {
+    let app: INestApplication;
+    let processor: ApplicationChangesReportIntegrationScheduler;
+    let db: E2EDataSources;
+    let sftpClientMock: DeepMocked<Client>;
+    let auditUser: User;
+    let sharedStudent: Student;
+
+    beforeAll(async () => {
+      const { nestApplication, dataSource, sshClientMock } =
+        await createTestingAppModule();
+      app = nestApplication;
+      db = createE2EDataSources(dataSource);
+      sftpClientMock = sshClientMock;
+      auditUser = await db.user.save(createFakeUser());
+      const fakeStudent = createFakeStudent();
+      sharedStudent = await saveFakeStudent(db.dataSource, {
+        student: fakeStudent,
+        sinValidation: createFakeSINValidation({ student: fakeStudent }),
+      });
```

Comment:
The `saveFakeStudent(db.dataSource)` created the student and also SIN.
Can it be simplified?